src/app/extensions/wishlists/facades/wishlists.facade.ts

```diff
@@ -69,8 +69,8 @@ export class WishlistsFacade {
     this.store.dispatch(addProductToNewWishlist({ title, sku }));
   }
 
-  addProductToWishlist(wishlistId: string, sku: string, quantity?: number): void {
```

Comment:
Why is the quantity attribute removed, although the property could be added as a query param to the addProductToWishlist REST call?

Reply:
https://support.intershop.com/kb/index.php/Display/29X616#l1162.4000244140625

Reply:
Afaik the `wishlists` is used both for order templates and wishlists. Order templates track quantities on them and you can also add a whole template to the basket. But I did not find any case where products are added to wishlists with quantities. Also, products are added individually from the wishlist to the basket with a quantity set there, [not from wishlist data](https://github.com/intershop/intershop-pwa/blob/7969ddabacf00bf027963b2faab7269829ce223b/src/app/extensions/wishlists/pages/account-wishlist-detail/account-wishlist-detail-page.component.html#L42-L47)

To be consistent, I removed all quantity handling from wishlists.

Reply:
Oh, just noticed, I did not yet remove it from the models.

Reply:
Done 😃

Reply:
Ok that makes sense :)